server/profileGet.php

```diff
@@ -8,9 +8,9 @@
   die();
 }
 require_once 'config.php';
-getProfile($USER->getUserId());
+getProfile($userId->getUserId());
```

Comment:
This change isn't what I meant, and as a matter of fact, this won't even work. What I meant was something like:
```
....
getProfile($USER->getUserId());
function getProfile($userId) {
.....
```